sui_types/src/unit_tests/base_types_tests.rs

```diff
@@ -169,6 +169,80 @@ fn test_object_id_serde_json() {
     assert_eq!(obj_id, json_obj_id);
 }
 
+#[test]
+fn test_object_id_serde_not_human_readable() {
```

Comment:
Currently object id is passed in the rest server in hex: https://github.com/MystenLabs/sui/blob/7c30c0a99102e46c47d7009d700e0617b60e7662/sui/src/rest_gateway/requests.rs#L57. Are we changing this behavior?

Reply:
I am not changing ObjectID in this PR, ObjectID already have it's own serialiser implemented earlier and it's doing similar things : Hex for human-readable, and tuple array (which doesn't contain array length compare to Bytes) for non-human-readable serialisation, we might want to do a refactoring in the future though to make them consistent.